packages/core/src/dispatcher.ts

```diff
@@ -4,6 +4,7 @@ import { StreamUtils, IpfsApi, UnreachableCaseError } from '@ceramicnetwork/comm
 import StreamID from '@ceramicnetwork/streamid'
 import { DiagnosticsLogger, ServiceLogger } from '@ceramicnetwork/common'
 import { Repository } from './state-management/repository'
+import { LruCache } from './state-management/lru-cache'
```

Comment:
We already use a library for an lruCache.

Reply:
😭

Reply:
I'll use the existing one.